docs/reference/config.md

```diff
@@ -673,7 +673,7 @@ The following settings are available:
 : Specifies Platform LSF *per-task* memory reserve mode. See {ref}`lsf-executor`.
 
 `executor.pollInterval`
-: Determines how often to check for process termination. Default varies for each executor (see below).
+: Determines how often to poll for process termination. Default varies for each executor (see below).
```

Comment:
Seems both not accurate. I'd say something like "Controls how often the system checks if the process has terminated" or "Defines the polling frequency for process termination detection"

Reply:
Good point. Suggestion applied.